target/linux/ath79/dts/qca9557_buffalo_bhr-4grv2.dts

```diff
@@ -103,15 +103,101 @@
 &mdio0 {
 	status = "okay";
 
-	phy0: ethernet-phy@0 {
+	switch0@10 {
+		compatible = "qca,qca8327";
+		#address-cells = <1>;
+		#size-cells = <0>;
+
 		reg = <0>;
+		qca,sgmii-rxclk-falling-edge;
+		ports {
+			#address-cells = <1>;
+			#size-cells = <0>;
+			port@0 {
```

Comment:
I know this is WIP, but in the final version one should introduce empty lines before the first port (port@0) and before the ports block.

Reply:
Noted.